docs/src/user-guide/configuring/configuration-files.md

```diff
@@ -359,7 +361,7 @@ module.exports = {
 
 ## Configuration Based on Glob Patterns
 
-<b>v4.1.0+.</b> Sometimes a more fine-controlled configuration is necessary, for example, if the configuration for files within the same directory has to be different. Therefore you can provide configurations under the `overrides` key that will only apply to files that match specific glob patterns, using the same format you would pass on the command line (e.g., `app/**/*.test.js`).
+<b>v4.1.0+.</b> Sometimes a more fine-controlled configuration is necessary, like if the configuration for files within the same directory has to be different. In this case, you can provide configurations under the `overrides` key that only apply to files that match specific glob patterns, using the same format you would pass on the command line (e.g., `app/**/*.test.js`).
```

Comment:
Should we change the `<b>` to markdown syntax for bold?

Reply:
yeah might as well while we're in here

Reply:
fixed